lhapdf5.sh

```diff
@@ -19,7 +19,7 @@ make install
 
 PDFSETS="cteq6l cteq6ll CT10 CT10nlo MSTW2008nnlo EPS09LOR_208 EPS09NLOR_208"
 pushd $INSTALLROOT/share/lhapdf
-  $INSTALLROOT/bin/lhapdf-getdata $PDFSETS
+  $INSTALLROOT/bin/lhapdf-getdata --repo="https://www.hepforge.org/archive/lhapdf/pdfsets/5.9.1" $PDFSETS
```

Comment:
I don't think we should hard code the version here

Reply:
I agree. Will "%(tag_basename)s" work here in the same way it is used at the start of the recipe?

Reply:
I made the repo link use the version number from the recipe tag.

Reply:
I think you'll need to strip it, e.g. ```${${version#v}%-*}```.

Maybe @dberzano has a better idea?